returnn/compile.py

```diff
@@ -228,12 +232,29 @@ def __init__(
         :param returnn_config: RETURNN config object
         :param checkpoint: Path to the checkpoint for export
         :param device: target device for graph creation
+        :param input_names: sequence of model input names.
```

Comment:
the order of parameters in function definition should match the order of the docstring (currently `device` is off)